openmoc/materialize.py

```diff
@@ -131,6 +131,9 @@ def load_from_hdf5(filename='mgxs.h5', directory='mgxs',
                 # the Material must be cloned for each unique Cell
                 if material != None:
                     if len(domains) > geometry.getNumMaterials():
+                        # Return memory ownership of material to Python
+                        # so it will be deallocated after material is cloned
+                        material.thisown = 1
```

Comment:
If I understand correctly, `thisown` can set to 0 or 1 signifying False and True. I feel like a boolean would make it a bit more obvious what is going on. What are your thoughts on setting `thisown` here (and perhaps other places) with booleans.